components/menu-dropdown/index.jsx

```diff
@@ -4,6 +4,7 @@
 // ### onClickOutside
 // Listen for clicks that occur somewhere in the document, outside of the element itself
 import onClickOutside from 'react-onclickoutside';
+import LanguageDirection from '../utilities/UNSAFE_direction/private/language-direction';
 import MenuDropdown from './menu-dropdown';
 
-export default onClickOutside(MenuDropdown);
+export default LanguageDirection(onClickOutside(MenuDropdown));
```

Comment:
Don't even have to wrap  `MenuDropdown` with `LanguageDirection`. 
`Dialog` is [wrapping itself](https://github.com/salesforce/design-system-react/blob/master/components/utilities/dialog/index.jsx#L525).